### PR TITLE
10-mig-project_department

### PR DESCRIPTION
**Pull request by thinkwelltwd**
Depends on https://github.com/OCA/account-analytic/pull/84

#171 hasn't been merged yet, but I needed this module. Hopefully you can use the PR.

### CODE REVIEW

**Comment by dreispt** [comment]
Dependencies are merged, I restarted the build.

**Review by dreispt — APPROVED** [review]

File: project_department/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Add Department to Projects and task to corresponding tree, search and form
+views.
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/project
```
Comment (LoisRForgeFlow) [inline project_department/README.rst]: this should be `https://runbot.odoo-community.org/runbot/140/10.0`

File: project_department/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+* Daniel Reis
```
Comment (LoisRForgeFlow) [inline project_department/README.rst]: no mail provided @dreispt ?

File: project_department/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (LoisRForgeFlow) [inline project_department/__init__.py:2]: Headers won't hurt here.

File: project_department/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Joël Grand-Guillaume (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Project Department Categorization",
+    "version": "10.0.1.0.0",
+    "author": "Camptocamp, Daniel Reis,Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline project_department/__manifest__.py:7]: just asking, is it fine to add individual authors here?

File: project_department/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from . import project
```
Comment (LoisRForgeFlow) [inline project_department/models/__init__.py:2]: headers

File: project_department/models/project.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    project_department_id = fields.Many2one(
+        related='project_id.department_id',
+        comodel_name='hr.department',
+        string='Project Department',
+        store=True,
```
Comment (LoisRForgeFlow) [inline project_department/models/project.py:14]: is this needed? wouldn't this be stored by default?

**Review by LoisRForgeFlow — COMMENTED** [review]
Some comments

File: project_department/__manifest__.py (hunk 4 above)
Comment (dreispt) [reply project_department/__manifest__.py:7]: Yes, if you're not doing the work hired by a company.

File: project_department/models/project.py (hunk 6 above)
Comment (dreispt) [reply project_department/models/project.py:14]: Its not stored by default, and afaicr it is needed for UI search.

**Comment by dreispt** [comment]
@lreficent Can we merge this?

File: project_department/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply project_department/README.rst]: @dreispt  @thinkwelltwd this is a fast fix, can you please attend and I will approve.

**Comment by LoisRForgeFlow** [comment]
see my new comment in the review: https://github.com/OCA/project/pull/253#discussion_r122389316

File: project_department/models/__init__.py (hunk 5 above)
Comment (dreispt) [reply project_department/models/__init__.py:2]: There was a discussion on that, and they are not required on `__init__.py` files

File: project_department/README.rst (hunk 1 above)
Comment (dreispt) [reply project_department/README.rst]: [Done](https://github.com/OCA/project/pull/253/commits/f1fd30aa7f11e2384718d4f3d7cadbce3d26af18)

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM now :+1:

**Pull request merged**